### PR TITLE
Introduce IProperty interface

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@decanus ^^

File: contracts/IProperty.sol [hunk 1]
```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.20;
+
+import "openzeppelin-solidity/contracts/token/ERC721/ERC721Full.sol";
+import "./IERC721WithCreator.sol";
+
+contract IProperty is ERC721Full, IERC721WithCreator {
```
Comment (axic) [inline contracts/IProperty.sol]: Needs to be `IERC721Full` or whatever is the appropriate one.

Comment (decanus) [inline contracts/IProperty.sol]: I'd say lets actually have it an ```interface``` and ugly import all the functions we need. When will solidity allow interface inheritance @axic?????

Comment (axic) [reply contracts/IProperty.sol]: There's an issue for it. Hopefully soon.

**Pull request merged**